### PR TITLE
[MWPW-137523] Add dark and large styles for quiet marquee

### PR DESCRIPTION
**Pull request by elan-tbx**
* As per the linked Jira ticket, a request was made to increase the max-width of text in `quiet` variant marquees with an additional `large` variant. Furthermore, min-heights were applied to the `quiet` marquee t-shirt sizes.
* Dark mode did not appear to work on `quiet` marquees. This PR addresses that.

Resolves: [MWPW-137523](https://jira.corp.adobe.com/browse/MWPW-137523)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/ebartholomew/quiet-marquee?martech=off
- After: https://ebartholomew-quiet-marquee-updates2--milo--adobecom.hlx.page/drafts/ebartholomew/quiet-marquee?martech=off

### CODE REVIEW

File: libs/blocks/marquee/marquee.css [hunk 1]
```diff
@@ -17,25 +17,35 @@
...
-.marquee.mobile-dark {
+.marquee:is(.mobile-dark, .quiet.dark) {
   color: var(--color-white);
 }
 
+.marquee .icon-area :is(picture, a) {
+  display: contents;
+}
+
+.marquee:is(.mobile-dark, .quiet.dark) a:is(:not(.con-button), :not(.con-button):hover) {
```
Comment (JasonHowellSlavin) [inline libs/blocks/marquee/marquee.css:32]: This is more compact, but I think less readable. What about: 
```
.marquee:is(.mobile-dark, .quiet.dark) a:not(.con-button),
.marquee:is(.mobile-dark, .quiet.dark) a:not(.con-button):hover {
```

File: libs/blocks/marquee/marquee.css [hunk 2]
```diff
@@ -356,13 +361,12 @@
     color: var(--link-color);
   }
 
-  .marquee.tablet-light a:not(.con-button):hover {
-    color: var(--link-hover-color);
+  .marquee.tablet-dark a:is(:not(.con-button), :not(.con-button):hover) {
```
Comment (JasonHowellSlavin) [inline libs/blocks/marquee/marquee.css:364]: same as above for me, not sure i like the `:is(:not(...))` pattern, but understand why.

**Review by JasonHowellSlavin — APPROVED** [review]
nits, otherwise looks good to me. 

The is:(:not()) in some areas made it hard to see that you were not removing styles that were there but just truncating them instead.

**Review by Sartxi — APPROVED** [review]
LGTM

**Pull request merged**